helpers/util.sh

```diff
@@ -170,18 +209,21 @@ buildmw() {
     #  -b     Branch to use. If none supplied, use default.
     #  -s     .spec file to use. Can be supplied multiple times.
     #         If empty, will use all .spec files from $PKG/rpm/*.
+    #  -N     Tell scratchbox to not fix the version inside a spec file.
 
     local GIT_URL=""
     local GIT_BRANCH=""
     local MW_BUILDSPEC=""
+    local NO_AUTO_VERSION
```

Comment:
@Thaodan, `local` means that `NO_AUTO_VERSION` will not inherit the value if it was passed as cmdline arg to `build_packages.sh`, but being able to rebuild any package without autofixing its version would be a good additional feature.

Reply:
It will as when you set it before in globally it will take this and put in to the local copy of the variable.

Reply:
I've run a minimum test case and it's not picking up the value:
```bash
#!/bin/bash

var=glob

f() {
  local var
  echo $var
}

f
```

Reply:
Than change it to local NO_AUTO_VERSION=$AUTO_VERSION this works as I said/it should.

Reply:
Changed to `local NO_AUTO_VERSION=$NO_AUTO_VERSION` with added comment, otherwise it'd blow code reader's mind:)